vars/skipStage.groovy

```diff
@@ -220,12 +220,12 @@ boolean call(Map config = [:]) {
                    prReposContains('leap15', jobName()) ||
                    skip_stage_pragma('build-leap15-rpm')
         case 'Build DEB on Ubuntu 20.04':
-            return paramsValue('CI_RPM_ubuntu20_NOBUILD', false) ||
+            return paramsValue('CI_DEB_Ubuntu20_NOBUILD', false) ||
                    target_branch =~ branchTypeRE('weekly') ||
                    (docOnlyChange(target_branch) &&
                     prRepos('ubuntu20') == '') ||
                    prReposContains('ubuntu20', jobName()) ||
-                   skip_stage_pragma('build-ubuntu20-rpm')
+                   skip_stage_pragma('build-ubuntu20-deb')
```

Comment:
This was intentionally left as `rpm` due to the documenting of it as such at https://daosio.atlassian.net/wiki/spaces/CI/pages/1389759093/Commit+Pragmas.  I have no inherent objection to this change but the documentation needs updating at least.

You could also allow for both postfixes here so that you don't trip up anyone that is using the existing documented pragma.

Ultimately this should have probably been `-pkg` to be generic.

Reply:
@brianjmurrell Do you think such a change (`-rpm` to `-pkg`) would be acceptable if both documentation and pipeline will be updated?

Reply:
I think we could deprecate `*-rpm` in favour of `-*pkg` by adding (not replacing -- yet) `*-pkg` as an additional condition and then updating the documentation to specify that `*-pkg` is not preferred and that `*-rpm` is deprecated.

Reply:
I propose to do this in a separate ticket as it requires more changes in several files.
I roll back the last commit and will create a new one dedicated to replacing `-rpm` with`-pkg`